packages/backend/src/projectAdapters/projectAdapter.ts

```diff
@@ -72,10 +73,13 @@ export const projectAdapterFromConfig = async (
                 // TODO add selector to dbt cloud
             });
         case DbtProjectType.GITHUB:
+            const githubToken = config.installation_id
```

Comment:
Do you think we should check the `config.authorization_method` ? 

If I change from oauth to PAT, the installation_id will still be populated.

Reply:
I believe I tested this change, and it was working fine because the installation_id input does not get rendered. But it is probably more readable and less confusing if we add the method directly. So I'll do it